UI/js-src/lsmb/DateTextBox.js

```diff
@@ -24,8 +24,6 @@ define([
                params.placeholder = lsmbConfig.dateformat;
             }
             // end of 'old code' support block
-
-            this.inherited(arguments);
```

Comment:
Why should the parent class of this widget *not* be initialized?

Reply:
Parent constructors are called automatically and we must not inherit within a constructor. Dojo reports an error with child classes.